core/spec/models/spree/promotion_spec.rb

```diff
@@ -709,10 +709,6 @@
 
           it { is_expected.to be false }
         end
-
-        context "and at least one item is promotionable" do
-          it { is_expected.to be true }
-        end
```

Comment:
Either way, we need to clarify or remove this spec as is testing "and the items are all non-promotionable"